.gitlab-ci.yml

```diff
@@ -55,7 +55,7 @@ stages:
     - RESOURCES="-t 30 -N 1"
     - echo -e "section_start:$(date +%s):dependencies\r\e[0K
       Build dependencies and generate host-config file (uberenv)"
-    - ${EXEC_PREFIX} ${RESOURCES} scripts/llnl/build_tpls.py --spec=${SPEC} --directory=${CI_PROJECT_NAME}
+    - ${EXEC_PREFIX} ${RESOURCES} scripts/llnl/build_tpls.py --spec=${SPEC} --directory=${CI_BUILDS_DIR}/serac_tpls/${CI_RUNNER_ID}
```

Comment:
is this in a directory that will be removed in the normal gitlab manner?

Reply:
I have to manually cron job clean up our gitlab stuff anyway.

Reply:
oh yea.. forgot that awesome feature of gitlab.

Reply:
FYI: putting "CI_RUNNER_ID" does NOT prevent concurrent access to this directory _even in the same pipeline_.
A single runner can handle up to 8 jobs.
At the same time, there are several runners for quartz, so this directory will end up being duplicated.

(note: I like commenting merged PRs ;))